colm/tableflux/call.c.in

```diff
@@ -0,0 +1,68 @@
+#include <colm/colm.h>
```

Comment:
It does not, but it cannot be checked in otherwise `go build` will complain about the spurious C file and also it should have write permission removed, so I just use the sedsubst for consistency.